[#1170] Update total contribution to reflect based on checked file types (#1171)
reposense/RepoSense: docs/UserGuide.md

* `Breakdown by file type` : toggles the contribution bar to either display the bar by :
    * the total lines of codes added (if checkbox is left unchecked), or
    * a breakdown of the number of lines of codes added to each file type (if checkbox is checked). If sorting criteria is contribution, only lines contributed to selected file types will be taken into account.

>     * the total contribution of each group will get updated based on the selected file type checkboxes, while their variance will not.

Reviewer: I think it would be better if this is combined into the 2nd point as the original doc emphasize the difference between checked and unchecked.

>If sorting criteria is contribution, only lines contributed to selected file types will be taken into account.

This sentence can be moved to bottom as remark or moved to the notes.

Author: I combined both the added sentence and 
> If sorting criteria is contribution, only lines contributed to selected file types will be taken into account.

as one,  placed them into the notes section.

This is because the `Breakdown by file type` was talking about how it affects contribution bar, so i think additional infos like these should be in the notes section.

* `Breakdown by file type` : toggles the contribution bar to either display the bar by :
    * the total lines of codes added (if checkbox is left unchecked), or
    * a breakdown of the number of lines of codes added to each file type (if checkbox is checked). If sorting criteria is contribution, only lines contributed to selected file types will be taken into account.

>     * the total contribution of each group will update based on changes in file type checkboxes, while their variance does not.

Reviewer 2: ```suggestion
    * the total contribution of each group will get updated based on the selected file type checkboxes, while their variance will not.
```
